metpy/calc/thermo.py

```diff
@@ -1205,3 +1205,43 @@ def _isen_iter(iter_log_p, isentlevs_nd, ka, a, b, pok):
 
     # output values as a list
     return ret
+
+
+@exporter.export
+@check_units('[pressure]', '[temperature]', '[temperature]', '[temperature]')
+def sbcape_cin(pressure, temperature, dewpt, profile=None):
```

Comment:
The idea with the profile argument is to allow passing in a profile that may have already been calculated for plotting purposes to avoid redundant calculations.